core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterDMTest.java

```diff
@@ -11600,6 +11627,22 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
         .ok(expectedSparkSql);
   }
 
+  @Test public void testForBitAndNotFunction() {
+    final RelBuilder builder = relBuilder();
+    final RexNode bitPart = builder.call(SqlLibraryOperators.BITANDNOT,
+        builder.literal(3), builder.literal(2));
+
+    final RelNode root = builder
+        .scan("EMP")
+        .project(builder.alias(bitPart, "Result"))
+        .build();
+
+    final String expectedQuery = "SELECT BITANDNOT(3, 2) AS Result\nFROM scott.EMP AS EMP";
```

Comment:
change this to `expectedDB2Query`

Reply:
@nosheenSheikh02 Chaged the name.Please have a look.